services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/BaseHanke.kt

```diff
@@ -13,19 +15,54 @@ interface BaseHanke : HasYhteystiedot {
 }
 
 interface HasYhteystiedot {
-    val omistajat: List<HankeYhteystieto>?
-    val rakennuttajat: List<HankeYhteystieto>?
-    val toteuttajat: List<HankeYhteystieto>?
-    val muut: List<HankeYhteystieto>?
+    val omistajat: List<Yhteystieto>?
+    val rakennuttajat: List<Yhteystieto>?
+    val toteuttajat: List<Yhteystieto>?
+    val muut: List<Yhteystieto>?
 
-    fun extractYhteystiedot(): List<HankeYhteystieto> =
+    fun extractYhteystiedot(): List<Yhteystieto> =
         listOfNotNull(omistajat, rakennuttajat, toteuttajat, muut).flatten()
 
-    fun yhteystiedotByType(): Map<ContactType, List<HankeYhteystieto>> =
-        mapOf(
-            ContactType.OMISTAJA to (omistajat ?: listOf()),
-            ContactType.RAKENNUTTAJA to (rakennuttajat ?: listOf()),
-            ContactType.TOTEUTTAJA to (toteuttajat ?: listOf()),
-            ContactType.MUU to (muut ?: listOf()),
-        )
+    fun yhteystiedotByType(): Map<ContactType, List<Yhteystieto>> {
+        return mapOf(
```

Comment:
Could be a matter of taste but the return could be omitted with =.

Reply:
I changed this and the other functions in this file to use the expression body syntax.